### PR TITLE
Adds sort: published_at to featured video sections

### PR DESCRIPTION
**Pull request by kellgauthier**
## Problem 
[Rally](https://rally1.rallydev.com/#/66096747656ud/custom/24109743995?detail=%2Fdefect%2F483959759580&fdp=true)
When I delete a featured tag on a video for music and add a new one, the most recent shows up last.... I would want this to show up first since it's the most recently featured item. 

## Solution 
Sorts videos with featured tag by published_at

### CODE REVIEW

**Review by silasbaur — DISMISSED** [review]
Need to add `| reverse`. Videos are currently showing oldest video first

File: videos/featured.html [hunk 1]
```diff
@@ -40,6 +40,7 @@ <h2 class="section-title text-white flush-bottom">Featured Videos</h2>
 
       {% for video in site.videos %}
```
Comment (silasbaur) [inline videos/featured.html:41]: Need to ` | sort: 'published_at' | reverse` here

File: index.html [hunk 2]
```diff
@@ -28,6 +28,7 @@ <h1 class="font-family-condensed-extra text-uppercase jumbotron-title">Crossroad
             <div class="d-none">{% increment slot %}</div>
             {% for video in site.videos %}
```
Comment (silasbaur) [inline index.html:29]: Need to add ` | sort: 'published_at' | reverse` here

**Review by silasbaur — DISMISSED** [review]

**Comment by kellgauthier** [comment]
@silasbaur - I corrected the liquid syntax, and checked by adding `{{ video.published_at | date: "%b %d, %y" }}` when building locally to verify it was newest first.

**Review by silasbaur — APPROVED** [review]

**Pull request merged**